.gitignore

```diff
@@ -6,3 +6,7 @@ dist
 .pnpm-debug.log
 .DS_Store
 tsconfig.tsbuildinfo
+
+# build artifacts
+apps/*/public/sitemap.xml
+apps/*/public/sitemap-[0-99].xml
```

Comment:
TIL

Reply:
This .gitignore pattern matching syntax was new to me, found it here https://www.golinuxcloud.com/gitignore-examples/#5_Examples_of_pattern_matching_in_gitignore

![CleanShot 2022-09-30 at 12 57 59@2x](https://user-images.githubusercontent.com/694063/193329287-713a2acf-b563-4042-9635-1fb433700c6c.png)